### DISCUSSION TITLE
LCD screen showing black even after flashing

### DISCUSSION THREAD

**Discussion by Oisin27** [Jyers/Marlin#2040] [Q&A]
Ive seen a lot of people commenting on this but their screen problems seem to be their screen not being able to flash, mines is able to flash but turns black when i try turn it on after to test it. Ive tried many different methods but nothing seems to be working. I would appreciate help, Ive not even got to use my printer yet as I tried doing the update as soon as i put it together.

**Comment by NeedsMoreDakka** [comment]
Have you tried flashing it again?

On Sun, Sep 18, 2022 at 9:34 AM Oisin27 ***@***.***> wrote:

> Ive seen a lot of people commenting on this but their screen problems seem
> to be their screen not being able to flash, mines is able to flash but
> turns black when i try turn it on after to test it. Ive tried many
> different methods but nothing seems to be working. I would appreciate help,
> Ive not even got to use my printer yet as I tried doing the update as soon
> as i put it together.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Jyers/Marlin/discussions/2040>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AVGP2X33XZFN6IWRSM3CUXLV64K4VANCNFSM6AAAAAAQPOLFNQ>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Success is counted sweetest
By those who ne'er succeed.
To comprehend a nectar
Requires sorest need.
  -Emily Dickinson 1859
